packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -18,6 +18,7 @@ import {
 
 /*
  * Generate configuration combinations for a particular compat version
+ * NOTE: Please update this packages README.md if the default versions and config combination changes
```

Comment:
If we put the default versions in a json file it will be more naturally self-documenting...

Reply:
Yep.  Try to just document the behavior with this change.  Feel free to open an issue or fix it.